### DISCUSSION TITLE
Connect Username with IP address

### DISCUSSION THREAD

**Discussion by mitolovesyou** [BUZZARDGTA/Session-Sniffer#6] [Ideas]
would be nice is there was another config file to link someone's username with their details. i get into the same lobby with many players and my plan is to record the last disconnected IP when somebody leaves and then put it on a list with their username (or a config file if you implement it) so if ever i get into the same session with that user, their username will pop up next to their details. when someone leaves, you've basically obtained their IP address since you can view the last recorded disconnected user. i just want to be able to put someone's username and IP on a config list and if they join the same session as me again, their username will be next to their IP and details instead of it says N/A. that's all thanks!

**Comment by BUZZARDGTA** [comment]
@mitolovesyou Hello, sorry for not giving you an answer earlier it seems I don't get notifications from discussions here.
Yes of course this is possible and it has been possibe for a long time already.
Please refer to [UserIP INI databases Configuration](https://github.com/BUZZARDGTA/GTA-V-Session-Sniffer/blob/main/docs/SCRIPT_CONFIGURATION.md#userip-ini-databases-configuration)

**Comment by BUZZARDGTA** [comment]
In v1.1.8, I realized I accidentally removed the code that creates the folder and default UserIP file when the script starts. I’ll fix this in v1.1.9 later.

For now, please create them manually to look like this:
![image](https://github.com/user-attachments/assets/fee5bd08-ccad-4053-ae4d-5b0471cffa32)

**Reply by mitolovesyou** [reply]
sounds good, thanks!

**Reply by jeyquaw** [reply]
> In v1.1.8, I realized I accidentally removed the code that creates the folder and default UserIP file when the script starts. I’ll fix this in v1.1.9 later.
> 
> For now, please create them manually to look like this: ![image](https://private-user-images.githubusercontent.com/62464560/383910549-fee5bd08-ccad-4053-ae4d-5b0471cffa32.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODQ3NzcwNzEsIm5iZiI6MTc4NDc3Njc3MSwicGF0aCI6Ii82MjQ2NDU2MC8zODM5MTA1NDktZmVlNWJkMDgtY2NhZC00MDUzLWFlNGQtNWIwNDcxY2ZmYTMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA3MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNzIzVDAzMTkzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNlZWEyMGNhOWY5MWYzNzM5NzA2MzFjNWJkOGY0ODI2MjM5NzdjYjVlNmViYTAzMDgzNmUxZGNiMTkxMTUzNTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.arDn94eQL4JAxyPRkRtIEVTQtI9cgFJ3lSdNrH_gPMY)

Do you have a file to connect usernames with ips now?

**Reply by BUZZARDGTA** [reply]
Always been there yeah

**Reply by jeyquaw** [reply]
Can you send it here?
